### ISSUE TITLE
command.run() uses absolute path instead of relative path

### ISSUE THREAD

**Issue by mho22** [open-southeners/vscode-laravel-pint#50]
The whole Laravel Pint code works with a `Project.php` class that will localize the project folder thanks to `Project::path()` that uses php `getcwd()` method under the hood.

If you run this command from your `/` path : 

`/path/to/project/vendor/bin/pint /path/to/project/file.php` even if you have a `pint.json` file in the project it won't take it into consideration and will format your `file.php` with the default `"preset" : "laravel"`.

That's why it is mandatory to use the relative path : 

`cd path/to/project`

`vendor/bin/pint file.php`

Actually within the `PintEditService.ts` file on lines 191...205 we have : 

```
    let command = workspaceFolder
      ? await this.moduleResolver.getPintCommand(workspaceFolder, file.fsPath)
      : await this.moduleResolver.getGlobalPintCommand([file.fsPath]);

    if (!command) {
      this.statusBar.update(FormatterStatus.Error);

      this.loggingService.logError(SOMETHING_WENT_WRONG_FINDING_EXECUTABLE + ' ' + pkg.bugs.url);

      return false;
    }

    command.run();

    this.loggingService.logDebug(RUNNING_PINT_ON_PATH, { command: command.toString() });
```

in debug mode it will output this : 

```
["DEBUG" - 15:53:57] Formatting file(s) using Pint
{
  "command": "/path/to/project/vendor/bin/pint --config /path/to/project/pint.json /path/to/project/file.php"
}
```

Thanks to the manually added `--config` the format can be done but the rest of the code using `Project::path()` won't work because it won't return `/path/to/project`.

The simple solution bould be to modify the file `PintEditService.ts` at line `203` from : 

```
command.run();
```

to
```
command.run( workspaceFolder?.uri.fsPath );
```

Can I possibly suggest a PR?

**Comment by d8vjork** [comment]
I see what you mean, go ahead with the PR but this we need to make sure that this also works on a multi-workspace environment

**Comment by mho22** [comment]
I made a PR. Could this be possible that the multi-workspace environment won't pose problem here thanks to the : 

```
const workspaceFolder = workspace.getWorkspaceFolder(file);
```
 
It returns `undefined` if the `file` doesn't match any workspace folder. but should return the file's workspaceFolder if the file is correct. I suppose one file can only have one workspaceFolder right ?

**Comment by d8vjork** [comment]
Should be fixed on this v1.1.6 👍
